include/game.h

```diff
@@ -1686,7 +1686,7 @@ extern s32 (*g_api_SetVolumeCommand22_23)(s16 vol, s16 distance);
 extern void (*g_api_func_800F53A4)(void);
 extern u32 (*g_api_CheckEquipmentItemCount)(u32 itemId, u32 equipType);
 extern void (*g_api_GetPlayerSensor)(Collider* col);
-extern void (*g_api_func_800F1FC4)(s32 arg0);
+extern void (*g_api_RevealSecretPassageAtPlayerPositionOnMap)(s32 arg0);
```

Comment:
I noticed this wasn't renamed after the `g_api` function was so have corrected. It only affected one place inside NO4 `e_secrets` so not a big impact